src/helpers/ipfs-daemon.js

```diff
@@ -76,4 +77,27 @@ const setIPFSCORS = (ipfsRpc) => {
   )
 }
 
-module.exports = { startIPFSDaemon, isIPFSCORS, setIPFSCORS }
+const isIPFSRunning = async (ipfsRpc) => {
+  const portTaken = await isPortTaken(ipfsRpc.port)
+
+  if (portTaken) {
+    if (!ipfsNode) ipfsNode = ipfsAPI(ipfsRpc)
+
+    const getId = promisify(ipfsNode.id)
+
+    try {
+      // if port is taken, attempt to fetch the node id
+      // if this errors, we can assume the port is taken
+      // by a process other then the ipfs gateway
+      await getId()
+      return true
+    } catch (e) {
+      console.log('error', e)
```

Comment:
I would suggest we get rid of this, as showing two errors to the users can be confusing:
```
✔ Start a local Ethereum network
 ❯ Check IPFS
   ⠙ Start IPFS
     Add local files
   Publish app to APM
   Create DAO
   Open DAO
error SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at streamToValue (/home/daniel/repos/aragon/aragon-cli/node_modules/ipfs-api/src/utils/stream-to-json-value.js:25:18)
    at concat (/home/daniel/repos/aragon/aragon-cli/node_modules/ipfs-api/src/utils/stream-to-value.js:12:22)
    at ConcatStream.<anonymous> (/home/daniel/repos/aragon/aragon-cli/node_modules/concat-stream/inde ✔ Start a local Ethereum network
 ❯ Check IPFS
   ✖ Start IPFS
     → Starting IPFS failed: Error: serveHTTPApi: manet.Listen(/ip4/127.0.0.1/tcp/5001) failed: list…
     Add local files
   Publish app to APM
   Create DAO
   Open DAO
 ✖ Starting IPFS failed: Error: serveHTTPApi: manet.Listen(/ip4/127.0.0.1/tcp/5001) failed: listen tcp4 127.0.0.1:5001: bind: address already in use
```

Reply:
oops, didn't mean to leave that in